emr-user-role-mapper-application/usr/install/al2/emr-user-role-mapper

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Overall change looks fine.
1. Can you point to more details on the testing. Links to clusters using AL1 and Al2?
2. Update the README.md like the following -

```
sudo initctl status emr-user-role-mapper
emr-user-role-mapper start/running, process 5403

```

Reply:
Tested on cluster j-J0L15ZKEP3LL (AL1) and j-2KQIYNZBUCXZQ (AL2). URM came up fine on both clusters.